src/Customer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+namespace Clang;
+
+class Customer {
+  public $accounts; // ArrayOfAccount
+  public $smsOptIn; // string
+  public $optIn; // string
+  public $id; // long
+  public $title; // string
+  public $lastname; // string
```

Comment:
```suggestion
  /**
   * @var string $firstname
   */
  public $lastname; // string
```

Reply:
@annuh I already thought about adding typing, which is nice to have. Although I want to stick closely to the original package. However, I tried to add typing -> https://github.com/aonghus-NL/ClangAPI/pull/2/files
But it seems this fails immediately because a lot of properties are optional but sadly not documented. So doing something like this: 
```
/**
* @var Account[]
*/
public array $accounts
``` 
will fail because it's optional, now I know this is optional so I can fix it easily by making it `public ?array $accounts` but this is not the only one, there are a lot that I don't know. 

Maybe we should just let it as it is, I'm also not sure if Clang has a newer package.

Reply:
> @annuh I already thought about adding typing, which is nice to have. Although I want to stick closely to the original package. However, I tried to add typing -> https://github.com/aonghus-NL/ClangAPI/pull/2/files
> But it seems this fails immediately because a lot of properties are optional but sadly not documented. So doing something like this:
> 
> ```
> /**
> * @var Account[]
> */
> public array $accounts
> ```
> 
> will fail because it's optional, now I know this is optional so I can fix it easily by making it `public ?array $accounts` but this is not the only one, there are a lot that I don't know.

It will also not work because CakePHP is using PHP 7.3 and doesn't support typed properties. Using PHP doc-blocks should always work.

> Maybe we should just let it as it is, I'm also not sure if Clang has a newer package.

You wanted to ask Clang if they had a new package right? I guess this is fine, some PHP doc-blocks would still be nice for the `Customer` resource :)

Reply:
Yeah I'm transforming it now to doc blocks!

Reply:
Btw looks like in CakePHP we are still using this file.
https://github.com/aonghus-NL/K2Go-Cake/blob/testing/app/Lib/Clang/clangAPI-api.myclang.com-1.22.php

There is nothing in the composer.json what mentions Clang...

Reply:
@annuh Updated it with doc blocks and made it less strict in the ClangAPI Class